subquery/src/mappings/grants.ts

```diff
@@ -131,7 +131,7 @@ export async function handleRenounced(event: RenouncedLog): Promise<void> {
 
   const action = new VestingScheduleRenounced(id, to, transaction.id);
 
-  const schedules = await VestingSchedule.getByCancelAuthorityId(from);
+  const schedules = await VestingSchedule.getByCancelAuthorityId(from, { limit: 100 });
```

Comment:
@Douglasacost This whole file seems to be obsolete looking at the latest deployed 
Grants contract. But I just made this fix for now to allow the build to work